packages/ui/src/molecules/Alert/stories/Alert.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import type { Story } from '@storybook/react'
+import React from 'react'
+
+import type { ComponentArgTypes } from '../../../typings/utils'
+import type { AlertProps } from '../Alert'
+import Component from '../Alert'
+import ShoppingCartIcon from '../../../atoms/Icon/stories/assets/ShoppingCart'
+import mdx from './Alert.mdx'
+
+const AlertTemplate: Story<AlertProps> = ({
+  testId,
+  dismissible,
+  icon,
+  children,
+  onClose,
+}) => (
+  <Component
+    testId={testId}
+    icon={icon}
+    dismissible={dismissible}
+    onClose={onClose}
+  >
+    {children}
+  </Component>
+)
+
+export const Alert = AlertTemplate.bind({})
+Alert.args = {
+  icon: <ShoppingCartIcon />,
+}
+
+export const AlertWithoutIcon = AlertTemplate.bind({})
+AlertWithoutIcon.args = {}
+
+export const AlertWithDismissButton = AlertTemplate.bind({})
+AlertWithDismissButton.args = { dismissible: true }
+
+export const AlertWithAction = AlertTemplate.bind({})
+AlertWithAction.args = {
+  children: (
+    <>
+      Alert <a href="/">Action</a>
+    </>
+  ),
+}
+
+const argTypes: ComponentArgTypes<AlertProps> = {
+  icon: {
+    control: { type: 'select' },
+    // eslint-disable-next-line react/jsx-key
+    options: [<ShoppingCartIcon />],
```

Comment:
This is [rendering](https://deploy-preview-1020--faststoreui.netlify.app/?path=/story/molecules-alert--alert) `[object Object]` in the `icon` select in the Controls tab:

![Zhi5cnTHEV](https://user-images.githubusercontent.com/381395/142483654-ac1a3d64-f23d-41cc-a91d-d06fe2ab52d4.gif)

Not sure how to fix it though. :(

Reply:
Nice catch! Let's fix it using [controls with complex values](https://storybook.js.org/docs/react/essentials/controls#dealing-with-complex-values).